build.js

```diff
@@ -0,0 +1,22 @@
+var rollup = require( 'rollup' );
+var babel = require( 'rollup-plugin-babel' );
+var replace = require( 'rollup-plugin-replace' );
+
+var env = process.argv[2] === '--prod' ? 'production' : 'development';
+
```

Comment:
how about `process.argv.contains('--prod') >= 0`, then it can be on any argument.